bedrock/firefox/templates/firefox/new/trailhead/thanks-a.html

```diff
@@ -0,0 +1,83 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% from "macros-protocol.html" import hero, picto_card with context %}
+
+{% extends "firefox/new/trailhead/base.html" %}
+
+{% block extrahead %}
+  {{ super() }}
+  {{ css_bundle('firefox_exp_thanks') }}
+{% endblock %}
+
+{% block site_header %}
+  {% include 'includes/protocol/navigation/menu-firefox/index.html' %}
+{% endblock %}
+
+{% block content %}
+<main role="main">
+  <div class="mzp-c-notification-bar mzp-t-warning unsupported">
+    <p>
+      {{ ftl('firefox-new-youre-using-an-insecure-outdated', url='https://support.mozilla.org/kb/end-support-windows-xp-and-vista') }}
+    </p>
+  </div>
+  <div class="mzp-c-notification-bar mzp-t-success auto-download">
+    {# fallback_url is replaced by the platform download link via JS, but if
+       something fails the user should still get a link to a working download path. #}
+    <p>
+      {{ ftl('firefox-new-your-firefox-new-should-begin', id='direct-download-link', fallback_url=url('firefox.all'))}}
+    </p>
+  </div>
+
+  {% call hero(
+    title='Great decision! Your download has started.',
+    desc='Follow the easy steps below to complete your switch.',
+    class='mzp-has-image mzp-t-product-firefox t-switch',
+    include_cta=True,
+    heading_level=1
+  ) %}
+
+    <div id="download-button-wrapper-desktop">
+      {{ download_firefox(force_direct=true, dom_id='primary-download-button') }}
+    </div>
+
+  {% endcall %}
+
+  <section class="c-steps mzp-l-content">
+    <ol class="c-steps-list">
+      <li class="c-steps-item">Install Firefox and open it up.</li>
+      <li class="c-steps-item">Follow <a href="https://support.mozilla.org/kb/import-bookmarks-google-chrome">this tutorial</a> to import your passwords, bookmarks, browsing history, and other data from Chrome.</li>
+      <li class="c-steps-item">Enjoy exploring the internet with more speed and privacy.</li>
+    </ol>
+  </section>
+
+  <section class="c-share">
+    <div class="mzp-l-content">
+      <h2 class="c-share-title">Spread the word about Firefox and help your favorite people say goodbye to Chrome.</h2>
+
+      {% set share_url = settings.CANONICAL_URL + canonical_path %}
```

Comment:
The share buttons are stil pointing to /firefox/download/thanks/ instead of /firefox/new/?